File: cpp codes/code11.cpp
#include <iostream>
#include <string>
using namespace std;

void team11() {
    string x = "15";
    int y = x + 5; // Logical Error: Cannot add string and integer
    cout << y << endl;
    cout << "You qualify for the next round!" << endl;
}

int main() {
    team11();
    return 0;
}

File: code_11.py
def fibonacci(n):
    a, b = 0, 1
    for _ in range(n):
        yield a
        a = b
        b = a + b  # Logical mistake: should be b = a + b, a = b.

# Main code
print("Fibonacci Series:", list(fibonacci(5)))  # Syntax mistake: incorrect function call brackets.

# End message
print("Congratulations! You have qualified for the next round.")
